docs/user-guide/cli-configure-daemon-on-zlinux-os.md

```diff
@@ -34,18 +37,22 @@ The following steps describe how to install and build the daemon binary on z/Lin
 
    After the command completes successfully, the Zowe daemon binary is a file named `zowe` that can be found in the `target/release` directory.
 
-7. Copy the binary to another location on the system and add it to your PATH.
-8. (Optional) Modify the file permissions to allow others to use the same binary:
+7. Copy the binary to another location on the system and add that location **[is "that location" correct?]** to your PATH.
```

Comment:
@adam-wolfe @zFernand0 - please respond

Reply:
"that location" is valid. Basically, they need to add the path to the directory where they copied the binary to their PATH environment var.

Reply:
`-7` Yes, "that location" is accurate since we want them to be able to find that binary in their path.